CMMS/Graph/Entry/WOOrderEntry.cs

```diff
@@ -134,6 +158,39 @@ protected virtual IEnumerable Schedule(PXAdapter adapter)
 
             return adapter.Get<WOOrder>();
         }
+        #endregion
+
+        #region Open Action
+        public PXAction<WOOrder> open;
+        [PXButton(CommitChanges = true), PXUIField(DisplayName = "Open", MapEnableRights = PXCacheRights.Select)]
+        protected virtual IEnumerable Open(PXAdapter adapter) => adapter.Get<WOOrder>();
+        #endregion
+
+        #region Complete Action
+        public PXAction<WOOrder> complete;
+        [PXButton(CommitChanges = true), PXUIField(DisplayName = "Complete", MapEnableRights = PXCacheRights.Select)]
+        protected virtual IEnumerable Complete(PXAdapter adapter) => adapter.Get<WOOrder>();
+        #endregion
+
+        #endregion
+
+        #region Events
+        #region WOOrder_RequestApproval_FieldDefaulting
+        protected void _(Events.FieldDefaulting<WOOrder.requestApproval> e)
+        {
+            e.NewValue = Setup.Current.WORequestApproval;
```

Comment:
Can we just make this an attribute on WOOrder.requestApproval?

[PXDefault(typeof(WOSetup.woRequestApproval))]

Reply:
Resolved in latest commit